src/app/researcher/study/request/[memberSlug]/actions.ts

```diff
@@ -20,6 +21,18 @@ export const onCreateStudyAction = researcherAction(async ({ memberSlug, studyIn
 
     const member = await getMemberFromSlugAction(memberSlug)
```

Comment:
I don't see where this is being called, but if you rename `memberSlug` to be `orgSlug` and call it like:
`onCreateStudyAction({ orgSlug, studyInfo })` 

`researcherAction` will now will automatically check that the user is a member of that organization